src/openzaak/components/zaken/openapi.yaml

```diff
@@ -3625,7 +3624,7 @@ components:
             NATUURLIJK PERSOON
           type: string
           maxLength: 17
-        inpA_nummer:
+        inpANummer:
```

Comment:
another breaking changes from the upstream standard -> check what's causing this and if we can prevent it